chore: Crowdin English suggestions

aleene opened the pull request:
**Description:**
The suggestions to english text changes from Crowdin.

aleene committed:
Crowdin suggestion

aleene committed:
Crowdin suggestion

aleene committed:
Typo

aleene committed:
Crowdin suggestions

aleene committed:
Crowdin suggestion

aleene committed:
crowdin suggestion

Source text should be amended to e.g.:

>> "1800+ contributors, like you, have added 13000+ products from 134 countries using the <0>Android</0>, <1>iPhone</1> or <2>Windows Phone</2> apps or cameras to scan barcodes and upload product pictures and labels.".

Obviously, non-breaking spaces (&nbsp;) *should never* be used for numbers formatting, as number formats differ in various target languages, and when this code is left out of the localization output, translators will see a completely irrelevant error message generated by the editor.
Instead, this type of separator should either be left out completely or replaced by the AE (,) or BE (.) equivalent.

I assume that the use of a space char as a numeric thousands separator is yet another French linguistic peculiarity, but this text is in English, meaning that French conventions are in no way not relevant here, sorry.

aleene committed:
Crowdin suggestion

Many source texts in this project contain redundant words/wordings.

Unfortunately, it seems that a greater focus has been placed on ensuring a heavy use of "we", "us" & "our" (that contribute no tangible user value) in as many of the source texts as possible rather than avoiding any unnecessary redundancies such as seen in this case, which generally is a great pity.

Suggest amending this source text to e.g.:

>> "The <0>search form</0> contains dozens of criteria against which you can match products.".

aleene committed:
Crowdin suggestion

Spaces are not used in front of semicolons:

>> "message:Do you like Open Food Facts?".

aleene committed:
crowdin suggestion

"...invent..." seems a less optimal choice of wording here (has already happened!), "...develop..."/"...further develop..." might be a better choice.

aleene committed:
Crowdin suggestion

>> "Open Products Facts is a general product database (excepting foods, cosmetics, and pet food) made by everyone, for everyone."
/
"Open Products Facts is a general product database (foods, cosmetics, and pet food exempted) made by everyone, for everyone.".

aleene committed:
Crowdin suggestion

>> "<0>Join us on our Slack</0> in the
#openpetfoodfacts channel to discuss and invent Open Pet Food Facts!".

hangy commented on lang/en/texts/data.html:
```diff
@@ -129,7 +129,7 @@ <h3>If your users expect a result immediately (e.g. Nutrition apps)</h3>
 <li>Send nutrition facts + category &gt; get Nutri-Score</li>
 <li>Send ingredients &gt; get the NOVA group (about food ultra-processing), additives, allergens, normalized ingredients, vegan, vegetarian&hellip;</li>
-<li>Send category + labels &gt; soon get the Eco-Score (about environmental impact)</li>
+<li>Submit category + labels &gt; soon get the Eco-Score (about environmental impact)</li>
```
I'm kind of wondering if we should spell out `+` and `&` instead. What do you think?

aleene replied:
Agree

hangy commented on lang/en/texts/press-and-blogs.html:
```diff
@@ -14,7 +14,7 @@ <h1>Press and blogs</h1>
 
 <li><a href="https://blog.okfn.org/2013/03/04/open-food-facts/">Open Knowledge Foundation blog</a> - 04/03/2013</li>
-<li><a href="https://www.slideshare.net/TonZijlstra/going-beyond-open-government-data">Going Beyond Open Government Data - Ton Ziljstra - European Data Week, Nantes (présentation)</a> - 25/05/2012</li>
+<li><a href="https://www.slideshare.net/TonZijlstra/going-beyond-open-government-data">Going Beyond Open Government Data - Ton Zijlstra - European Data Week, Nantes (présentation)</a> - 25/05/2012</li>
```
Presentation should be without an accent in English.

aleene committed:
Crowdin suggestion

>> "And as a consequence, it has to
be public, free, and open.".

(Error correction + optimization.)

aleene committed:
Crowdin suggestion

Or simply (i.e. less for users to read):

>> "Manufacrurer or brand name".

aleene committed:
Crowdin suggestion

replaced all computed with calculated

aleene committed:
Crowdin suggestion

So, what's the exact intended meaning here:

Products that have abbreviations associated with their names (as indicated by the current source text)
or
Product names that appear in abbreviated form, i.e. "Abbreviated product names as shown on receipts"?

These two wordings clearly indicate different conditions.

aleene committed:
Crowin suggestion

Again, probably yet another instance of erroneous (in software/related environments) use of "we", as it's assumed that no staff involvement is really necessary/required in this process/context.

Using pronouns in software (environments) doesn't meet the same goals/objectives as is the case with hardcopy material, on the contrary, in fact.

aleene committed:
Crowdin suggestion

>> "The product doesn-t exist yet in the public database"
/
"The product is not yet in the public database".

aleene committed:
Crowdin suggestions

+1 for the above:

>> "Manufacturers can use the Open Food Facts <a href="<producers_platform_url>">free producers platform</a> to access and complete this data, and to obtain reports, analysis and product improvements hints (e.g. better Nutri-Score).".

Not clear what "...opportunities..." is supposed to signify/convey here.

aleene committed:
Crowdin suggestion

Instead, most likely just ("Type of the product..." just sounds strange):

>> "Product photo type".

Alternatively (probably what was meant all along, which actually makes sense as well):

>< "Type of product photo info"
/
"The type of product photo info".

aleene committed:
Crowdin suggestion

NCAA
Assume that there's an error here, specifically the use of "...experimented...", which seems out of place in this context:

>> "Removing or modifying provided data is only possible for experienced site contributors."
/
"Removing or modifying provided data is only possible for seasoned site contributors.".

aleene
Should this be authorised users (which happen to be experienced)?

aleene committed:
Crowdin suggestion

NCAA
@openfoodfacts:
>> "Submit...".

aleene committed:
Crowdin suggestion

aleene committed:
Crowdin suggestion

NCAA
@openfoodfacts:
>> "...by submitting an...".

aleene committed:
Crowdin suggestion

NCAA
@openfoodfacts:
>> "The photo is being uploaded…".

aleene committed:
Crowdin suggestion

NCAA
"...tightly cropped..." refers to what specifically?

aleene committed:
Crowdin suggestion

>> "Zoom in on...".

"Zoom" is just the description of aspecific feature, and these days, a widely used vido conference platform.

aleene committed:
Crowdin suggestion

For aforementioned reasons, "...we..." serves no purpose here:

>> "...is known, it's displayed under...".

aleene committed:
Crowdin suggestion

Unnecessarily extensive text (robs people's time):

>> "To sustain and expand our activities, we'll be deploying full-time paid human resources dedicated to the project.".

aleene committed:
Crowdin suggestion

Doesn't make sense having "brands, and nutritional composition etc." where "and" should be removed, i.e.:
"brands, nutritional composition etc."

aleene committed:
Crowdin suggestion

Incorrect use of "&nbsp;", as there is no space char placed before "!", "?", ":", ";" etc., which naturally includes "No break space" as well.

How many more times yet must this simple fact be pointed out?

aleene committed:
Crowdin suggestion

Since a question is asked in a sentence here, proper writing style dictates the use of an ending/concluding "?", which is currently missing.

And just to nip the potential of an error in the butt, no space char must be placed before the "?", not even a "No break space" code!

aleene committed:
Crowdin suggestion

The sentence seems to be incorrect somehow and the meaning is quite incomprehensible

aleene committed:
Crowdin suggestion

These are still not massive nor highly complicated calculations:

"...computed?" >> "...calculated?".

aleene committed:
Crowdin suggestion

...and what are these rather strange wordings "... in the apps of other actions..." & "...who built the Eco-Score." supposed to mean in clear text?

aleene committed:
Crowdin suggestion

No need for not using the abbreviated form OFF + the "we" part serves no purpose as no staff involvement is required for this process (calculation):

>> "The more than 100 applications that use the OFF database will also be able to display the calculated Eco-Score.".

aleene committed:
Crowdin suggestion

Considering the commendable wish to scale the OFF project into that a global initiative/movement, this continuous use of only domestic references for all presented instances still makes absolutely no sense. This just points to a narrow-minded, non-global viewpoint & general mindset out of sync with the otherwise expressed wishes for the future of this project/initiative.
Do the required legwork, dig up the relevant numbers, and use global locations/regions/different countries instead of globally indifferent domestic numbers when referring to various types of stats.

Why OFF believes that the average consumers located in e.g. Asia will give a hoot about being presented with, to them, irrelevant French data is truly a conundrum. If, in fact, they're interested in data from other regions, then they'll just search explicitly for such data, they don't need to get it forced down their throats unsolicited.

This overall fact pertaining to narrow-mindedness has been pointed out more than once already.

aleene committed:
Crowdin suggestion

aleene committed:
Normalized

Normalized to the spelling Eco-Score

aleene committed:
Crowdin

What "...the offer..."?

aleene committed:
Crowdin issue

aleene committed:
Crowdin suggestion

As pointed out previously, the exact meaning of "...position" is not clear.

aleene
This is the title of the image: https://world-en.openfoodfacts.org/files/presskit/PressKit/advanced-search/search-position.png. I think "Product search field" is a better name.

aleene committed:
Crowdin suggestion

And again, "...play on the color..." refers to what exactly?

aleene committed:
Crowdin suggestions

How many more times will it be necessary to point out that using space chars before e.g. ":", ";", ".", "!", "?" *is incorrect*, before this annoying and continuously recurring erroneous practice is finally put down like a rabies-infected dog and buried for good?
Since this project uses English as its source language, and *not* French, then please annihilate all non-English malformations from the source texts.
It's unbelievable that the exact same types of errors have to be pointed out/flagged again and again and again in what resembles an endless loop, and yet nothing is done to correct these errors, let alone ensure that they don't reoccur.
This indifference towards error correction/prevention and QA of texts is predominantly encountered in certain Chinese-based projects--and now in the OFF project too, unfortunately.

If eradicating this specific and otherwise fairly simple error type is such an insurmountable problem/task, then write a validation script for handling this overall condition since Crowdin apparently has no tools that can do this (...or could it simply be that said tools just aren't used?).

aleene committed:
Update data.html

aleene committed:
Update press-and-blogs.html

stephanegigandet commented on lang/en/texts/eco-score-the-environmental-impact-of-food-products.html:
```diff
@@ -2,7 +2,7 @@ <h1>Eco-Score: the environmental impact of food products</h1>
 <img src="https://static.openfoodfacts.org/images/icons/ecoscore-a.svg" alt="Eco-Score">
 
-<p><i>The Eco-Score is an environmental score (ecoscore) from A to E which makes it easy to compare the impact of food products on the environment.</i></p>
+<p><i>The Eco-Score is an environmental score from A to E which makes it easy to compare the impact of food products on the environment.</i></p>
```
that one is for SEO purposes in fact :)

aleene replied:
OK, revert

stephanegigandet commented on lang/en/texts/eco-score-the-environmental-impact-of-food-products.html:
```diff
@@ -209,13 +209,13 @@ <h2>How the Eco-Score can encourage the improvement of products and the reductio
 </ul>
 
-<p>This second lever is very important because it allows an overall improvement of the offer and thus reaches all consumers.</p>
+<p>Manufacturers improving the Eco-Score of their products is the ultimate goals, as it will be good for all consumers.</p>
```
```suggestion
<p>Manufacturers improving the Eco-Score of their products is the ultimate goal, as it will benefit all consumers.</p>
```

stephanegigandet commented on lang/obf/en/texts/press-and-blogs.html:
```diff
@@ -16,13 +16,13 @@ <h1>Press and blogs</h1>
+<li><a href="https://linuxfr.org/news/open-beauty-facts-que-contiennent-vraiment-nos-produits-cosmetiques">Open Beauty Facts : what do our cosmetics really contain?</a> (in french) -  31/07/2016</li>
+<li><a href="https://twitter.com/Teolemon/status/909453116889817089">Open Beauty Facts, a french crowdsource champion</a> (in french)
+<li><a href="https://twitter.com/RatZillaS/status/837251555657076736">Open Beauty Facts - Phosphore</a> (in french)
+<li><a href="https://twitter.com/Teolemon/status/776776806565285888">Open Beauty Facts - Practical Linux</a> (in french)
```
Linux Pratique is a magazine name, I wouldn't translate it.

aleene replied:
OK

stephanegigandet commented on po/common/common.pot:
```diff
@@ -106,8 +106,8 @@ msgid "This is a producer or brand account."
 msgctxt "producer_or_brand"
-msgid "Name of producer or name of brand"
-msgstr "Name of producer or name of brand"
+msgid "Manufacturer or brand name"
```
we should add a comment:
# "Name of" is repeated on purpose to make sure producers actually put their name of the name of their brand,
# and do not put "Manufacturer" or "Brand name" in the field. It happened multiple times.

aleene replied:
That does not help in Crowdin though

aleene replied:
I think the issue was also "Name of …" versus "… name" (french versus english)

stephanegigandet commented on po/common/common.pot:
```diff
@@ -5279,8 +5279,8 @@ msgid "Abbreviated product name"
-msgid "Product name with abbreviations shown on receipts"
-msgstr "Product name with abbreviations shown on receipts"
+msgid "Abbreviated product name as shown on receipts"
+msgstr "If a consumer buys a product in a shop, he will get a payment ticket. The product name on the ticket will be abbreviated."
```
msgstr should be empty, is this a comment?

aleene replied:
It was not empty, the one above as well? I'll add it as a comment (not useful for Crowdin).

stephanegigandet commented on po/common/common.pot:
```diff
@@ -5296,8 +5296,8 @@ msgid "Forgotten password?"
-msgid "We could not reinitialize your password, please contact us for assistance."
-msgstr "We could not reinitialize your password, please contact us for assistance."
+msgid "Your password could not be reset, please contact us for assistance."
+msgstr "Message shown if the password could not be reset by the system."
```
msgstr should be empty, is this a comment?

aleene replied:
The msgstr has data everywhere. Should this be changed throughout the file?

teolemon replied:
common.pot is the template, so msgstr should be empty in it.

teolemon replied:
`.pot` = po template as opposed to `.po`

stephanegigandet reviewed (changes requested):
Overall it looks very good, but there are a few things we want to keep as they were. I made comments for those.

aleene committed:
Update eco-score-the-environmental-impact-of-food-products.html

aleene committed:
Update lang/en/texts/eco-score-the-environmental-impact-of-food-products.html

Co-authored-by: Stéphane Gigandet <stephane@openfoodfacts.org>

aleene committed:
Update press-and-blogs.html

aleene committed:
Update common.pot

aleene committed:
Update common.pot

aleene committed:
New update

Sorry, but both texts have unnecessary redundancy; two instances of "Eco-Score" is just overkill in such a short paragraph.

In addition, the precise, intended meaning is obfuscated thanks to the very strange wording of this source text.

Also, the 'Eco-Score launching partners' part in your suggestion doesn't seem accurate, as the launch of this rating system, as far as I know, has already happened, and, in addition, it might not be all the parties who helped along in the build process who are actually 'partners' in the literal sense of this word.

So, instead, suggest amending to something along the lines of:

>> "Eco-Score rating is/will soon be available in the apps of the market actors who helped build it."
/
"Eco-Score rating is/will soon be available in the apps of the market actors who helped devise it."
/
"The Eco-Score rating system is/will soon be available in the apps of the market actors who helped build it."
/
"Eco-Score rating is available/will soon be presented in the apps of the market actors who helped build it."
/
"Eco-Score rating is available/will soon be accessible in the apps of the market actors who helped build it."
/
"If not already implemented, Eco-Score rating is soon to be available in the apps of the market actors who helped devise it."
/
"If not already implemented, the Eco-Score rating system is soon to be available in the apps of the market actors who helped devise it."
/
etc.

So, no shortage of options liberated from this needless (and not visually appealing) redundancy... :-)

stephanegigandet reviewed (approved):
Looks good to me, thank you!

teolemon reviewed (changes requested):
- [ ] We have a new repository for the static files. https://github.com/openfoodfacts/openfoodfacts-web/tree/main/lang/en/texts
- [ ] I did a PR on April 17th to remove those files from here, since I have moved them to openfoodfacts-web https://github.com/openfoodfacts/openfoodfacts-server/pull/5158

aleene commented:
@teolemon Do you mean that the changes here are no longer relevant?

aleene committed:
Merge branch 'main' into aleene-english-crowdin

aleene commented:
This is getting old. Time to commit?

teolemon commented:
Heavily conflicted, and we moved lang to https://github.com/openfoodfacts/openfoodfacts-web
Can we close @aleene ?

aleene commented:
Guess we can, i am not going through things again
